src/components/application_manager/include/application_manager/rpc_handler_impl.h

```diff
@@ -161,17 +161,15 @@ class RPCHandlerImpl : public RPCHandler,
                           smart_objects::SmartObject& output);
   std::shared_ptr<Message> ConvertRawMsgToMessage(
       const ::protocol_handler::RawMessagePtr message);
-  hmi_apis::HMI_API& hmi_so_factory();
-  mobile_apis::MOBILE_API& mobile_so_factory();
 
   ApplicationManager& app_manager_;
   // Thread that pumps messages coming from mobile side.
   impl::FromMobileQueue messages_from_mobile_;
   // Thread that pumps messages coming from HMI.
   impl::FromHmiQueue messages_from_hmi_;
 
-  hmi_apis::HMI_API hmi_so_factory_;
-  mobile_apis::MOBILE_API mobile_so_factory_;
+  ns_smart_device_link_rpc::V1::v4_protocol_v1_2_no_extra
+      v4_protocol_so_factory_;
```

Comment:
@IGapchuk Absoletely unclear variable name

Reply:
@AByzhynar Fixed in c73ed4e0d89041779c761d4fc1f9820fe7c5ae94.